frontend/public/components/utils/resource-log.tsx

```diff
@@ -239,6 +324,27 @@ export const ResourceLog: React.FC<ResourceLogProps> = ({
     }
   }, [status, totalLineCount, previousTotalLineCount, bufferSize]);
 
+  //Check to see if previous log exists
+  React.useEffect(() => {
+    const podUrl = resourceURL(modelFor(resource.kind), {
+      name: resource.metadata.name,
+      ns: resource.metadata.namespace,
+      path: '',
+    });
+
+    axios
+      .get(podUrl)
+      .then((response) => {
+        const container = response.data.status.containerStatuses?.find(
+          (pod) => pod.name === containerName,
+        );
+        setPreviousLogs(container?.restartCount > 0);
+      })
+      .catch(() => {
+        setPreviousLogs(false);
+      });
+  }, [containerName, resource]);
```

Comment:
Hey, @kdoberst. We have a `k8sGet` utility to get a Kubernetes resource. In most places, however, we watch the resource instead using WebSockets with `useK8sWatchResource` or `Firehose` so that the UI updates when the resource updates.

The `resource` prop itself should be the `Pod` on the Pod page, so in this case, you shouldn't need to fetch it again. I believe that `resource` can be a `Build`, however, so we'll need to handle that case. @TheRealJon is the most familiar with the log code and can probably help.

Reply:
Thanks for pointing out that the content is already available.  Removing axios get call.